### DISCUSSION TITLE
ORA-64219: invalid LOB locator when storing Data Map

### DISCUSSION THREAD

**Discussion by suhasghorp** [quartznet/quartznet#1089] [General] [answered]
I am getting ORA-64219: invalid LOB locator encountered error when storing my Job data map into Oracle store.
One of the keys I am storing has a large string value of about 2500 characters. 
Is there any limit on the key-value store for data map ?
By trial and error, I have determined that I can store strings up to about 1600 chars in length but not more.

Thanks

**Comment by lahma** [comment]
What version of Quartz? What kind of configuration, OracleDelegate in use?

**Comment by suhasghorp** [comment, marked as answer]
Quartz 3.2.4 

yes, Oracle delegate in use. My config looks like

`<add key="quartz.jobStore.driverDelegateType" value="Quartz.Impl.AdoJobStore.OracleDelegate, Quartz" />`
`<add key="quartz.dataSource.QuartzStore.provider" value="OracleODPManaged" />`
`<add key="quartz.jobStore.type" value="Quartz.Impl.AdoJobStore.JobStoreTX, Quartz" />`

As it turns out, this error was related to using an older Oracle managed driver. 
When I upgraded to the latest 19.10.1, the error went away.

Thanks.
